dashboard/app/models/script_level.rb

```diff
@@ -115,12 +115,6 @@ def self.add_script_levels(script, lesson_group, lesson, raw_script_levels, coun
         sl.levels = levels
       end
 
-      # Generate and store the seed_key, a unique identifier for this script level which should be stable across environments.
```

Comment:
Why did you remove this?

Reply:
I'm not sure yet if I should actually remove this. please see https://github.com/code-dot-org/code-dot-org/pull/38145#discussion_r539854584

Reply:
sorry, I meant https://github.com/code-dot-org/code-dot-org/pull/38145#discussion_r537936192 .